src/App.jsx

```diff
@@ -1,28 +1,50 @@
 import React from 'react';
+
 import './App.scss';
 import './styles/general.scss';
+
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
+import { getTodos } from './api/todos';
+
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodos()
+      .then(todos => this.setState({ todos }));
+  }
+
+  selectUser = (userId) => {
+    this.setState({ selectedUserId: userId });
+  }
+
+  removeUser = () => this.setState({ selectedUserId: 0 })
+
   render() {
     const { todos, selectedUserId } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList todos={todos} />
+          <TodoList
+            todos={todos}
+            selectedUser={selectedUserId}
```

Comment:
```suggestion
```
You are passing `selectedUser` to the `TodoList` component but never use it there